src/components/icon/icon.tsx

```diff
@@ -28,11 +28,22 @@ export class Icon {
     @Prop({ reflectToAttr: true })
     public badge: boolean;
 
+    /**
+     * SVG data to use. Supply *instead* of `name` to show the supplied data,
+     * instead of loading icon from disk.
+     */
+    @Prop({ reflectToAttr: true })
+    public svgData: string;
```

Comment:
I'm not sure if we want to expose this possibility. I did as a first step, since that's the least complicated. If we don't want to expose it to the consumers of lime-elements, we can probably split the component, so we can keep the possibility of supplying raw svg data private, or at the very least undocumented.

I didn't want to create a completely separate new component though, because I wanted it to use the same css and stuff, regardless of where the svg data comes from.

Any input appreciated!